x/crosschain/types/status.go

```diff
@@ -86,3 +86,11 @@ func stateTransitionMap() map[CctxStatus][]CctxStatus {
 	}
 	return stateTransitionMap
 }
+
+func (c CctxStatus) IsTerminal() bool {
+	return c == CctxStatus_Aborted || c == CctxStatus_Reverted || c == CctxStatus_OutboundMined
+}
+
+func (c CctxStatus) IsPending() bool {
```

Comment:
Is it a good idea to just return `!c.IsTerminal` to indicates the two mutually exclusive states?